src/server/services/checkWebsiteStatus.js

```diff
@@ -0,0 +1,52 @@
+const cron = require('node-cron');
+const { getConnection } = require('../db');
+const { logMailer } = require('../logMailer');
+const LogEmail = require('../models/LogEmail');
+const colors = require('colors');
+  
+async function checkWebsite() {
+  const conn = getConnection();
+
+  try {
+    const response = await fetch('https://httpstat.us/500/', { method: 'HEAD' });
```

Comment:
What would be the url for production? is it https://openenergydashboard.org/index.html ?

Reply:
The URL is unique for each site. Thus, the side needs to be able to provide. If we do this via a cron script (see other comment) then it could be a value set in that script file. An alternative is to all it to be passed via arguments to the script but I don't know how off the top of my head for npm scripts.

Reply:
Oh I thought there one only one URL we are verifying. If that is the case. how about using command-line arguments like below? 
Then users can type in the unique url after the cron command, the system will recognize the url as the second argument and store it as WEBSITE_URL variable and verify its status

command example:
0 * * * * src/server/services/checkWebsiteStatus.js https://example.com >> /path/to/logfile.log 2>&1


![image](https://github.com/user-attachments/assets/6daafb9e-228e-4d1f-b995-7ace6606adad)

Reply:
That is fine. In the OED base scripts to run cron jobs we often have values set at the top to provide local information. That is fine but it is also okay to use the command line arg if it is present and default to the one in the script if not.